README.md

```diff
@@ -38,7 +38,9 @@ Install [tilt prerequisites](https://cluster-api.sigs.k8s.io/developer/tilt.html
 
 ## Running CAPC without Tilt - detailed instructions
 
-Generally speaking, this cloudstack infrastructure provider will generate a docker image and 3 yaml files. `clusterctl` (a binary tool) will use the above docker image and 3 yaml files to provision a cluster from your local machine using cloudstack as a provider.
+Generally speaking, this cloudstack infrastructure provider will generate a docker image and 3 yaml files. 
+`clusterctl` (a binary tool) will use the above docker image and 3 yaml files to provision a cluster from your local 
+machine using cloudstack as a provider.
```

Comment:
Largely cleaned description to reflect v1beta1 and wrapped some very long lines.